<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Trevor Lancon</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>740</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>barbellcv</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,0,0,1" columnstretch="0,0,1" rowminimumheight="0,0,0,0,0,1">
    <item row="1" column="2">
     <widget class="QTableWidget" name="tableSetStats"/>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QGridLayout" name="layoutMarkerColor" columnstretch="1,0,0,0">
      <item row="5" column="0">
       <widget class="QLabel" name="labelMarkerDiameter">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Marker Diameter</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" rowspan="2">
       <widget class="QLabel" name="labelWeight">
        <property name="text">
         <string>Weight</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelExercise">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Exercise</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEditLogPath"/>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="buttonSaveSettings">
        <property name="text">
         <string>Save Settings</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinMinHue">
        <property name="maximum">
         <number>180</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelHue">
        <property name="text">
         <string>Hue</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelValue">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelSaturation">
        <property name="text">
         <string>Saturation</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinMinValue">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinMinSaturation">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboRotation"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelCamera">
        <property name="text">
         <string>Camera</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="Line" name="lineSep1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelRotation">
        <property name="text">
         <string>Camera Rotation</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="4">
       <widget class="Line" name="lineSep2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QComboBox" name="comboExercise"/>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="QPushButton" name="buttonPreview">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" rowspan="3">
       <widget class="QPushButton" name="buttonSelectColor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Select Color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="comboCamera"/>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="spinLbs">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1102.299999999999955</double>
        </property>
        <property name="value">
         <double>45.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinDiameter">
        <property name="maximum">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QPushButton" name="buttonOpenSet">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Open Set</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="labelKgs">
        <property name="text">
         <string>kgs</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="spinKgs">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>501.000000000000000</double>
        </property>
        <property name="value">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinMaxValue">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="labelLbs">
        <property name="text">
         <string>lbs</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinMaxHue">
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="value">
         <number>180</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSpinBox" name="spinMaxSaturation">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="labelMarkerUnits">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEditLifter">
        <property name="text">
         <string>Lifter</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="labelLifter">
        <property name="text">
         <string>Lifter</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3" rowspan="3">
       <widget class="QPushButton" name="buttonLogSet">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Log Set</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QCheckBox" name="checkSaveVideo">
        <property name="text">
         <string>Save video</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="2" rowspan="2">
     <widget class="PlotWidget" name="plotTimeline" native="true"/>
    </item>
    <item row="4" column="0" rowspan="2" colspan="2">
     <widget class="PlotWidget" name="plotMotion" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExportToCSV"/>
    <addaction name="actionRefreshCameraList"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="menuCamera">
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionSave_settings">
   <property name="text">
    <string>Save settings</string>
   </property>
  </action>
  <action name="actionClear_saved_videos">
   <property name="text">
    <string>Clear saved videos</string>
   </property>
  </action>
  <action name="actionExportToCSV">
   <property name="text">
    <string>Export database to CSV...</string>
   </property>
  </action>
  <action name="actionRefreshCameraList">
   <property name="text">
    <string>Refresh camera list</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboCamera</tabstop>
  <tabstop>comboRotation</tabstop>
  <tabstop>buttonPreview</tabstop>
  <tabstop>spinMinHue</tabstop>
  <tabstop>spinMaxHue</tabstop>
  <tabstop>spinMinSaturation</tabstop>
  <tabstop>spinMaxSaturation</tabstop>
  <tabstop>spinMinValue</tabstop>
  <tabstop>spinMaxValue</tabstop>
  <tabstop>buttonSelectColor</tabstop>
  <tabstop>spinDiameter</tabstop>
  <tabstop>buttonSaveSettings</tabstop>
  <tabstop>spinLbs</tabstop>
  <tabstop>spinKgs</tabstop>
  <tabstop>comboExercise</tabstop>
  <tabstop>lineEditLifter</tabstop>
  <tabstop>checkSaveVideo</tabstop>
  <tabstop>buttonLogSet</tabstop>
  <tabstop>lineEditLogPath</tabstop>
  <tabstop>buttonOpenSet</tabstop>
  <tabstop>tableSetStats</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
